test/zepto.html

```diff
@@ -2175,6 +2175,14 @@
           $.map(el.children(), function(c){ return $(c).text() }).join(' '))
       },
 
+      testAppendWithArrayParams: function(t){
+        var node = document.createElement('b');
+        node.appendChild(document.createTextNode('four'))
+        var el = $('<div></div>')
+        el.append(['<b>one</b><b>two</b>', '<b>three</b>', node])
+        t.assertEqual('<b>one</b><b>two</b><b>three</b><b>four</b>', el.html())
```

Comment:
Scratch that; my testing shows that jQuery supports all of these cases. So, each argument to `append()` can be a DOM node, jQuery object, HTML string, or an array holding any mixture of those types. So, my initial comment is valid; we should add a test that some array items can be Zepto collections.